plugins/node/opentelemetry-instrumentation-mysql/src/instrumentation.ts

```diff
@@ -323,6 +332,11 @@ export class MySQLInstrumentation extends InstrumentationBase<
     };
   }
 
+  private _defaultDbStatementSerializer(dbStatement: string) {
+    //replace all quoted words with a question mark (' and ". not `)
+    return dbStatement.replace(/['"]([^'"]*)['"]/g, '?');
+  }
```

Comment:
a better place for it is in `utils.ts` IMO.

This function belongs to the class but it is not using `this` and is more a utility than instrumentation object method